A52-xds-custom-lb-policies.md

```diff
@@ -0,0 +1,312 @@
+A52: gRPC xDS Custom Load Balancer Configuration
+----
+* Author(s): temawi
+* Approver: markdroth
+* Implemented in: Java, Go and C-core
```

Comment:
Hasn't been implemented in Core yet :)

Reply:
`Once a proposal is Final and if it has been implemented by a language, it can be updated to a status of Implemented with the implementing languages listed. (Listing versions is not required.)`

Reply:
Ok, I removed the languages for now.